fs/notify/fanotify/fanotify_user.c

```diff
@@ -826,7 +827,6 @@ SYSCALL_DEFINE2(fanotify_init, unsigned int, flags, unsigned int, event_f_flags)
 	if (fd < 0)
 		goto out_destroy_group;
 
-	group->fanotify_data.flags = flags;
```

Comment:
why move this line?

Reply:
ok, that is more correct although pid/tid is not relevant for the FAN_OVERFLOW event
I have folded this change into my patch